dye/tasklib/database.py

```diff
@@ -225,6 +225,9 @@ def test_root_password(self, password):
         return self.test_sql_user_password(user='root', password=password)
 
     def test_grants(self):
+        if not self.grant_enabled:
+            return
+
```

Comment:
Should this be `return True` rather than just `return` (which implicitly returns `None` which is effectively `False` )